.github/workflows/docs.yaml

```diff
@@ -1,64 +0,0 @@
-on:
```

Comment:
Why was the docs test deleted? this seems like it may be unintended

Reply:
It is by mistake (the original branch this branch created had that commit. Re-introduced it. Thanks for catching it)